app/components/ItunesCard/index.js

```diff
@@ -0,0 +1,188 @@
+      if (audioRef.current) {
+        const fraction = (audioRef.current.currentTime / audioRef.current.duration) * 100;
+        setProgress(fraction);
+      }
+    }, 1000);
+    setIntervalStore(intervalValue);
+  }
+
+  const handlePlaying = (val) => {
+    if (val) {
+      audioRef.current.play();
+      calculateProgress();
+    } else {
+      audioRef.current.pause();
+    }
+    handleOnActionClick(audioRef.current);
+  };
+
+  return (
+    <CustomCard
+      data-testid="itune-card"
+      hoverable
+      title={trackName}
+      actions={[
+        <If condition={artistViewUrl} key="artistViewUrl">
+          <Tooltip title={intl.formatMessage({ id: 'view_artist' })}>
+            <Button
+              data-testid="view-artist"
+              type="link"
+              target="_blank"
+              href={artistViewUrl}
+              icon={<UserOutlined />}
+            ></Button>
+          </Tooltip>
+        </If>,
+        <If condition={trackViewUrl} key="trackViewUrl">
+          <Tooltip title={intl.formatMessage({ id: 'view_track' })}>
+            <Button
+              data-testid="view-track"
+              type="link"
+              target="_blank"
+              href={trackViewUrl}
+              icon={<AudioOutlined />}
+            ></Button>
+          </Tooltip>
+        </If>,
+        <If condition={collectionViewUrl} key="collectionViewUrl">
+          <Tooltip title={intl.formatMessage({ id: 'view_collection' })}>
+            <Button
+              data-testid="view-collection"
+              type="link"
+              target="_blank"
+              href={collectionViewUrl}
+              icon={<LinkOutlined />}
+            ></Button>
+          </Tooltip>
+        </If>
+      ]}
+    >
+      <If condition={artistName}>
+        <T id="artist_name" data-testid="artist-name" values={{ artistName: truncate(artistName, { length: 20 }) }} />
+      </If>
+      <If condition={collectionPrice && currency}>
+        <T id="collection_price" data-testid="collection-price" values={{ collectionPrice, currency }} />
+      </If>
+      <If condition={country}>
+        <T id="country" values={{ country }} data-testid="country" />
+      </If>
+      <If condition={collectionName}>
+        <T
+          id="collection_name"
+          data-testid="collection-name"
+          values={{ collectionName: truncate(collectionName, { length: 20 }) }}
+        />
+      </If>
+      <ButtonDiv>
+        <CustomButton onClick={() => handlePlaying(true)} data-testid="play-event">
+          <PlayCircleFilled />
+        </CustomButton>
+        <CustomButton onClick={() => handlePlaying(false)} data-testid="pause-event">
+          <PauseCircleFilled />
+        </CustomButton>
+      </ButtonDiv>
+      <CustomAudio ref={audioRef} src={previewUrl} data-testid="audio-elem" type="audio/mpeg" />
+      <CustomProgress data-testid="progress-bar" success={{ percent: progress }} showInfo={false} status="active" />
```

Comment:
We could ignore `showInfo` here unless its default value is `true`

Reply:
Yeah it's default value is true.